ui/containers/WorkspaceDetails/index.tsx

```diff
@@ -97,17 +98,27 @@ interface Props extends RouteComponentProps<DetailsRouteProps> {
   changeWorkspaceOwner: () => void;
   modifyDiskQuota: () => void;
   modifyCoreMemory: () => void;
+  setOKButtonProps: (obj: any) => void;
 }
 
 class WorkspaceDetails extends React.PureComponent<Props> {
   public delayedFetchUsers = throttle((v: string) => {
     this.props.getUserSuggestions(v);
   }, 2000);
 
+  public getOKButtonProps = (v: string) => {
+    const _return = { disabled: true };
```

Comment:
@shafi1379 We are using camelCase naming convention for variables' naming. I'd like you to name variables with camelCase naming convention. I know why you used `_` for the suffix, but it seems better to use camelCase in naming convention.